relay-general/src/store/normalize/span/tag_extraction.rs

```diff
@@ -331,11 +331,16 @@ fn truncate_string(mut string: String, max_bytes: usize) -> String {
     string
 }
 
+const ALLOWED_SQL_ACTION: &str =
+    "SELECT|INSERT|DELETE|UPDATE|SET|SAVEPOINT|RELEASE SAVEPOINT|ROLLBACK TO SAVEPOINT";
+
 /// Regex with a capture group to extract the database action from a query.
 ///
-/// Currently, we're only interested in `SELECT`, `INSERT`, `DELETE` and `UPDATE` statements.
-static SQL_ACTION_EXTRACTOR_REGEX: Lazy<Regex> =
-    Lazy::new(|| Regex::new(r#"(?i)(?P<action>(SELECT|INSERT|DELETE|UPDATE))"#).unwrap());
+/// Currently we have an explicit allow-list of database actions considered important
+static SQL_ACTION_EXTRACTOR_REGEX: Lazy<Regex> = Lazy::new(|| {
+    let action_regex: String = format!("(?i)(?P<action>({}))", ALLOWED_SQL_ACTION);
+    Regex::new(&action_regex).unwrap()
+});
```

Comment:
I would rather have the all defined `allowed` action directly in the regex, which personally for me is easier to grasp, instead of jumping from one variable to another. 

Is there a reason why this was split? Can we merge everything in one line of the `Regex::new` again?

Changing the docs on the `SQL_ACTION_EXTRACTOR_REGEX` would be enough, or even rename it to something including `_ALLOWED` if that makes the regex usage even clearer.

Reply:
> Is there a reason why this was split

For git blame reasons - so that it becomes easier to tell if it was just the regex changing, or if it was functionality in `SQL_ACTION_EXTRACTOR_REGEX` changing.

If you feel strongly though I can change it back. Thoughts?

Reply:
I ended up inlining the regex in [d17203c](https://github.com/getsentry/relay/pull/2385/commits/d17203c9559e353b57463cec8611245a9ade169e), so changed.